iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/LoadXml.java

```diff
@@ -45,28 +49,93 @@ protected void writeRootElementAttributes(Load l, VoltageLevel vl, NetworkXmlWri
         writePQ(null, l.getTerminal(), context.getWriter());
     }
 
+    @Override
+    protected void writeSubElements(Load load, VoltageLevel parent, NetworkXmlWriterContext context) throws XMLStreamException {
+        IidmXmlUtil.runFromMinimumVersion(IidmXmlVersion.V_1_10, context, () -> writeModel(load, context));
```

Comment:
Is there a way with existing utility function to warm if not minimum version?

Reply:
Maybe we should evolve IidmXmlUtil.runFromMinimumVersion to add another "else" callback?

Reply:
Good idea! For now, I think we only have`runUntilMaximumVersion`...

Reply:
Not clear to me how to mix it with `runFromMinimumVersion`. Could you show me ?

Reply:
You can see it in tie line for example
```java
if (activePowerLimits1.isPresent()) {
            IidmXmlUtil.assertMinimumVersion(ROOT_ELEMENT_NAME, ACTIVE_POWER_LIMITS_1, IidmXmlUtil.ErrorMessage.NOT_NULL_NOT_SUPPORTED, IidmXmlVersion.V_1_5, context);
            IidmXmlUtil.runFromMinimumVersion(IidmXmlVersion.V_1_5, context, () -> writeActivePowerLimits(1, activePowerLimits1.get(), context.getWriter(),
                    context.getVersion(), context.isValid(), context.getOptions()));
        }
```

Here it will need a bit of re-writing but something like:
```java
load.getModel.ifPresent(model -> {
IidmXmlUtil.assertMinimumVersion(ROOT_ELEMENT_NAME, "model", IidmXmlUtil.ErrorMessage.NOT_NULL_NOT_SUPPORTED, IidmXmlVersion.V_1_10, context);
IidmXmlUtil.runFromMinimumVersion(IidmXmlVersion.V_1_10, context, () -> writeModel(model, context));
}
...
private static void writeModel(LoadModel model, NetworkXmlWriterContext context) {
...
}
````

Reply:
thanks, done